### ISSUE TITLE
[CLI] `bb login` should say it's going to open a webpage before doing so

### ISSUE THREAD

**Issue by brentleyjones** [buildbuddy-io/buildbuddy#6469]
**Is your feature request related to a problem? Please describe.**

Currently `bb login` immediately opens a web page. This is jarring. `gh auth login` waits for the user to press Enter before opening the webpage:

```shellsession
$ gh auth login
? What account do you want to log into? GitHub.com
? What is your preferred protocol for Git operations on this host? SSH
? Upload your SSH public key to your GitHub account? /Users/brentley.jones/.ssh/id_ed25519.pub
? Title for your SSH key: GitHub CLI
? How would you like to authenticate GitHub CLI? Login with a web browser

! First copy your one-time code: XXXX-XXXX
Press Enter to open github.com in your browser...
````

**Describe the solution you'd like**

Something this this output:

```shellsession
$ bb login
We need your personal API key. Please log in to https://app.buildbuddy.io/settings/cli-login and copy your API key, creating one if needed.
Press Enter to open https://app.buildbuddy.io/settings/cli-login in your browser...
```

**Comment by sluongng** [comment]
I think letting the user know and trigger the open with Enter makes a lot of sense. It's a great UX improvement and makes the CLI more predictable.

I am adding initial support for it in https://github.com/buildbuddy-io/buildbuddy/pull/6472/. I will add the full message in a different PR so that folks could comment on the exact content to be displayed.

Great suggestion!

**Comment by bduffany** [comment]
Closing since Son fixed this with #6472
